### ISSUE TITLE
data of step in steps before loop is cleared in v0.125.0

### ISSUE THREAD

**Issue by hayashikun** [k1LoW/runn#1128]
following runbook fails in v0.125.0, but not v0.124.1

```yaml
desc: some test
runners:
  req: http://localhost:8888
steps:
  step1:
    req:
      /hoge:
        get:
          body: null
    test: current.res.status == 404
  step2:
    req:
      /fuga:
        get:
            body: null
    test: current.res.status == 404
  step3:
    loop: 10
    req:
      /piyo:
        get:
          body: null
  step4:
    test: steps.step1 != null
  step5:
    test: steps.step2 != null  # fails because steps.step2 is null
```

I found that data of the step (`step2` of the above case) in the `steps` before the `loop` (`step3`) is cleared.

```
$ go install github.com/k1LoW/runn/cmd/runn@v0.125.0
$ runn run tmp/test.yaml
F

1) tmp/test.yaml edbd3b5e8edaa83ed75359e998103b79f6796c5c
  Failure/Error: test failed on "some test".steps.step5: condition is not true
  
  Condition:
    steps.step2 != null
    │
    ├── steps.step2 => null
    └── null => null
    
  Failure step (tmp/test.yaml):
  25   step5:
  26     test: steps.step2 != null


1 scenario, 0 skipped, 1 failure

```

**Comment by k1LoW** [comment]
@hayashikun Thank you for your report! I'll check it.

**Comment by k1LoW** [comment]
Fixed at [v0.125.1](https://github.com/k1LoW/runn/releases/tag/v0.125.1). Thank you!
